week3/homework/Program.cs

```diff
@@ -0,0 +1,201 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World, this is 3rd exercise!");
+
+app.MapGet("/ctf", () =>
+{
+    var temp = new Temperature(75);
+    var result = temp.CelsiusToFarenheit();
+    System.Console.WriteLine($"{temp.Degrees} celsius is equal {result} Farenheit");
+    var result1 = temp.FarenheitToCelsius();
+    System.Console.WriteLine($"{temp.Degrees} Farenheit is equal {result1} celsius");
+});
+
+app.MapGet("/cc", () =>
+{
+    var amount = -100;
+    var exchangeRate = new ExchangeRate("EUR", "DKK");
+    exchangeRate.Rate = 7.5m;
+    Console.WriteLine($"{amount} {exchangeRate.FromCurrency} is {exchangeRate.Calculate(amount)} {exchangeRate.ToCurrency}");

```

Comment:
same as previous, plus it would be interesting if you can pass the amount as an endpoint parameter.